src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -601,9 +603,10 @@ def load_ssh_private_key(
     return private_key
 
 
-def serialize_ssh_private_key(
+def _serialize_ssh_private_key(
     private_key: _SSH_PRIVATE_KEY_TYPES,
-    password: typing.Optional[bytes] = None,
+    password: typing.Optional[bytes],
```

Comment:
Is it possible to get `None` except by a direct call to an API that's private? `private_bytes` always sets it to an empty byte string or the password so it seems like we could just claim it's always bytes.